conf/ocsci/tainted_worker_nodes_ui.yaml

```diff
@@ -0,0 +1,3 @@
+---
+DEPLOYMENT:
+    ocs_operator_nodes_to_taint: 3
```

Comment:
Not sure if we would need a new yaml for tainted worker nodes. Cant we use 
https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocsci/tainted_worker_nodes.yaml ?

Reply:
I want to test only taint node [no infra nodes]